ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -235,6 +235,9 @@ library
                        Ouroboros.Consensus.Storage.LedgerDB.DiskPolicy
                        Ouroboros.Consensus.Storage.LedgerDB.HD
                        Ouroboros.Consensus.Storage.LedgerDB.HD.BackingStore
+                       Ouroboros.Consensus.Storage.LedgerDB.HD.DiffSeq
```

Comment:
Can we make this module `consensus`-agnostic, or is this unnecessary abstraction? Specifically, should we make a general-purpose diff sequence datatype?